### DISCUSSION TITLE
Automate browsers patching for users

### DISCUSSION THREAD

**Discussion by luxysiv** [JingMatrix/ChromeXt#71] [Show and tell]
Can you release assets for non-root user on Github release? I will make bash script to patch on Github Action

**Comment by luxysiv** [comment]
Like this : https://github.com/luxysiv/ChromeXt-patch

**Comment by JingMatrix** [comment]
If I understand correctly, do you want me to put `LSPatch` in the `ChromeXt` release?

Is that Okay, if I add releases of LSPatch and you use your bash script to download them from the `LSPatch` project?

Maybe it is better not to mix `ChromeXt` with `LSPatch`.

**Comment by JingMatrix** [comment]
I added the releases of LSPatch in its project: https://github.com/JingMatrix/LSPatch/releases

It is Okay?

**Comment by luxysiv** [comment]
Need ChromeXt-sign.apk and you should release lspatch.jar with version. I will make a cron check version . If new , it'll patches. It's only mod from my repository to patch Revanced

**Comment by JingMatrix** [comment]
Maybe you can check versions by looking at the release tags. I think LSPatch is quite stable, usually one doesn't need to update it. Android 14 is supported as well.

I am not familar with Revanced, but you cannot download files from two repos in your bash script?

**Comment by luxysiv** [comment]
Yes, It can. Thanks for your answers

**Comment by luxysiv** [comment]
https://github.com/luxysiv/ChromeXt-patch/releases/tag/all
What do you think about this

**Comment by luxysiv** [comment]
https://github.com/luxysiv/ChromeXt-patch/actions/runs/5540536462/jobs/10112752557

I can make it patch Vivaldi, Chome beta and Bromite... Maybe more. It's only a demo

**Comment by JingMatrix** [comment]
The patching command looks fine for me, keep working on it :smile: .
It will be helpful to your users if you can verify APKs downloaded from apkmirror are not touched.
It will be helpful to ChromeXt project if you find a way to test if the patched APKs work as expected.

**Comment by luxysiv** [comment]
It works fine. Like you patch by manager. APKmirror is safe to download

**Reply by hellboy1123** [reply]
"Hello Community,
I have been working on a website related to [Stick War Legacy ](https://stickwarapk.download/)and recently added a section where users can download mod APK files for the game. As a developer, I want to ensure that my website follows best practices and adheres to legal and ethical guidelines.
I would like to know if there are any specific considerations or guidelines I should be aware of when publishing a website that offers modded APK files. Are there any restrictions or potential legal issues that I should take into account? Additionally, what measures can I implement to ensure the security and safety of users who download mod APKs from my website?
I appreciate any advice or insights you can provide on this matter.
Thank you,

**Comment by luxysiv** [comment]
I have finished my repository. Now it can patch Microsoft Edge , Bromite, Vivaldi and Chrome bete. I checked. Works fine
